internal/attestation/azure/snp/validator.go

```diff
@@ -214,18 +214,18 @@ func (v *Validator) checkIDKeyDigest(ctx context.Context, report *spb.Attestatio
 	// the MAA if necessary.
 	switch v.config.FirmwareSignerConfig.EnforcementPolicy {
 	case idkeydigest.MAAFallback:
-		v.log.Infof(
+		v.log.Info(fmt.Sprintf(
```

Comment:
As mentioned above, I'd advocate for using fields here instead of the formatter. This doesn't need to be done within this PR though.